account_invoice_line_complimentary/models/res_company.py

```diff
@@ -0,0 +1,13 @@
+#  Copyright 2023 Simone Rubino - TAKOBI
+#  License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class ResCompany (models.Model):
+    _inherit = 'res.company'
+
+    complimentary_account_id = fields.Many2one(
+        comodel_name='account.account',
+        string="Complimentary Account",
```

Comment:
I think we don't need the string here

Reply:
:wave:
Thanks for having a look!
You are right, I have removed it.